api/app/src/command/medical/patient/append-doc-query-progress.ts

```diff
@@ -67,8 +67,10 @@ export async function appendDocQueryProgress({
     }
 
     const convert = documentQueryProgress.convert;
-    if (convertibleDownloadErrors && convert) {
+    if (convert && convertibleDownloadErrors != null && convertibleDownloadErrors > 0) {
       convert.total = Math.max((convert.total ?? 0) - convertibleDownloadErrors, 0);
+      // since we updated the total above, we should update the status as well
+      convert.status = getStatusFromProgress(convert);
```

Comment:
culprit of https://metriport.slack.com/archives/C04GEQ1GH9D/p1687286701451619?thread_ts=1687283150.337459&cid=C04GEQ1GH9D